comms/src/transports/tcp.rs

```diff
@@ -240,16 +201,15 @@ mod test {
     #[test]
     fn configure() {
         let mut tcp = TcpTransport::new();
-        tcp.set_send_buffer_size(123)
-            .set_recv_buffer_size(456)
-            .set_nodelay(true)
-            .set_ttl(789)
-            .set_keepalive(Some(Duration::from_millis(100)));
-
-        assert_eq!(tcp.send_buffer_size, Some(123));
-        assert_eq!(tcp.recv_buffer_size, Some(456));
+        // tcp.set_send_buffer_size(123)
+        //     .set_recv_buffer_size(456)
+        tcp.set_nodelay(true).set_ttl(789);
+        // .set_keepalive(Some(Duration::from_millis(100)));
+
+        // assert_eq!(tcp.send_buffer_size, Some(123));
+        // assert_eq!(tcp.recv_buffer_size, Some(456));
```

Comment:
remove?

Reply:
This (and all the others in this file) could come back - tokio doesn't support these socket options at the moment. I'd rather leave them in